src/System.ServiceModel.Http/src/System.ServiceModel.Http.builds

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="12.0" DefaultTargets="Build" xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <ItemGroup>
+    <Project Include="System.ServiceModel.Http.csproj" >
```

Comment:
Yes that is likely correct. If you can verify that the resulting binary for each configuration is the same I agree you should collapse them.